### DISCUSSION TITLE
Add support for task list system tray indicator

### DISCUSSION THREAD

**Discussion by wada3n** [alainm23/planify#1623] [Ideas]
Hi,
**Describe the solution you'd like**
what about adding a Task indicator in the wingpanel for elementary, or other distro like a tray icon and  without opening planner and re-closing it and returning to the main app i use, just an indicator to see my items within the panel to focus squarely on my to do lists and tasks.
**Describe alternatives you've considered**
This picture shows my suggestion
![DfqPgllXUAAxdlB](https://user-images.githubusercontent.com/13304792/81488854-97d72400-9266-11ea-8bec-5a5ebdc186eb.png)
Thank you very much.

**Comment by alainm23** [comment]
Thanks for commenting, I'm not sure if flatpack supports Wingpanel Indicators but if this is possible it would be a great feature.

**Comment by dmitry** [comment]
@alainm23 I'm a mint user and I'd like to see the same feature (in the system tray). Here is sample of adding icons and menu to the system tray in vala: https://gist.github.com/milisarge/386d2fb152518c3fc32bbf47d04aa954
And it's also related to the #384

**Comment by wada3n** [comment]
@alainm23 Thank you,You are doing a great job.
Yes flatpak supports that
https://docs.flatpak.org/en/latest/desktop-integration.html

**Comment by nyx-lyb3ra** [comment]
Hi, what Flatpak supports is the status icons through libappindicator, but adding support for a wingpanel indicator is a different thing. elementary OS 5 has dropped the support for the `wingpanel-indicator-ayatana`, so no support for status icons.

In order add the support for this feature, it would be necessary to use the API for create wingpanel indicators in elementary and use the API for libappindicators for any other DE with support for status icons.

Also, as elementary will soon migrate the whole App Center to Flatpak... I dunno if the wingpanel indicators will still working :thinking:

**Comment by ruanformigoni** [comment]
Hi, for those who use gnome, I made a [shell-extension](https://gitlab.com/formigoni/planner-gnome-tray-extension) to show planner on the system-tray, and open the application once clicked.

**Comment by Suzie97** [comment]
https://github.com/elementary/wingpanel-indicator-datetime/pull/252, elementary tasks will probably implement this in the near future. Can planner do the same thing as a third party app?? 🤔
